src/main/include/Robot.h

```diff
@@ -137,6 +138,8 @@ class Robot : public frc::TimedRobot {
         // Shooter camera
         photon::PhotonCamera a_camera{SHOOTER_CAMERA_NAME};
 
+        Vision a_Vision;
+
         frc::AprilTagFieldLayout aprilTagFieldLayout = frc::LoadAprilTagLayoutField(frc::AprilTagField::k2024Crescendo);
```

Comment:
If this is all encapsulated in the Vision class now, this would be safe to delete from the Robot class.